app/controllers/candidate_interface/applying_controller.rb

```diff
@@ -0,0 +1,20 @@
+module CandidateInterface
+  class ApplyingController < CandidateInterfaceController
+    rescue_from ActionController::ParameterMissing, with: :render_not_found
+
+    def show
+      provider_code = params.fetch(:providerCode)
+      course_code = params.fetch(:courseCode)
+
+      @course = FindAPI::Course.fetch(provider_code, course_code)
```

Comment:
Do we feel happy letting this guy out into the view unwrapped and naked? e.g. If Find change their API structure then it will ripple into our app.

Reply:
Have added a presenter 🚀